src/core/types/models.ts

```diff
@@ -18,6 +18,8 @@ export interface BaseModelConfig {
     messages: ChatMessage[],
     contextLimit: number,
   ) => Promise<ContextMetrics>;
+  contextWarningThresholdPercentage: number;
+  conversationResetThreshold: number;
```

Comment:
Adding these as model-specific in case we want to adjust limits more specifically